dependencies.cfg

```diff
@@ -4,3 +4,4 @@
 #   separator: colon + optional white space
 #   version: git branch or tag
 integrations.library: v0.6.0
```

Comment:
I've just release v0.7.0
Please update and test if there's no conflict.

Reply:
done